src/screens/FundingScreen.tsx

```diff
@@ -77,23 +81,18 @@ export class FundingScreen extends React.Component<Props, State> {
     const erroringTransactions = []
 
     if (wallet) {
-      for (const boostTransaction of this.state.boostTransactions) {
-        try {
-          await checkLNPayRecipientRoute(wallet, boostTransaction.normalizedValueRecipient)
-        } catch (error) {
-          if(error?.response?.data?.status === 400) {
-            erroringTransactions.push({address: boostTransaction.normalizedValueRecipient.address, 
-                                       message: error.response.data.message})
-          }
-        }
-      }
+      const { boostTransactions, streamingTransactions } = this.state
+      const { boostAmount, streamingAmount } = this.global.session.valueTagSettings.lightningNetwork.globalSettings
+      const transactionsToTest = boostAmount > streamingAmount ? boostTransactions : streamingTransactions
```

Comment:
If the recipients are the same, do we need to check one or the other? Can we just check the streaming ones?

Reply:
I originally thought we had to check whichever was the largest amount, but now that TimK has said it's basically fine to send 1 sat transactions, I'm just going to use the boost value for the queryroutes here.